app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,51 @@
+class VendorsController < ApplicationController
+  before_action :set_vendor, only: %i[show destroy]
+
+  # GET /vendors or /vendors.json
+  def index
+    @vendors = Vendor.all
+  end
+
+  # GET /vendors/1 or /vendors/1.json
+  def show
+  end
+
+  # POST /vendors or /vendors.json
+  def create
+    @vendor = Vendor.new(vendor_params)
+
+    # NOTE:  this is boilerplate from rails generator
+    # respond_to do |format|
+    #   if @vendor.save
+    #     format.html { redirect_to vendor_url(@vendor), notice: "Vendor was successfully created." }
+    #     format.json { render :show, status: :created, location: @vendor }
+    #   else
+    #     format.html { render :show, status: :unprocessable_entity }
+    #     format.json { render json: @vendor.errors, status: :unprocessable_entity }
+    #   end
+    # end
```

Comment:
Can you say more about why you included some of the boilerplate (line 15) but not the commented-out stuff?

Reply:
It's not clear what we will want.  This may be helpful to the developer doing the implementation.  If it isn't it can be removed.

Reply:
🆗